docs/ReleaseNotes.md

```diff
@@ -23,8 +23,39 @@ Place release notes for the upcoming release below this line and remove this lin
 
 ### Version 1.8.2505
 
+#### Potentially breaking changes
+
 - Typed buffers (including ROV buffers) no longer accept types other than vectors and scalars. Any other types will produce descriptive errors. This removes support for appropriately sized matrices and structs. Though it worked in some contexts, code generated from such types was unreliable.
-- By default, the internal validator will be used instead of searching externally for an existing DXIL.dll.
+  - Load and Store operations have been refactored as a consequence. Behavior should be identical, please file issues if discrepancies are observed.
+- The compiler will now always use the internal validator instead of searching for an external DXIL.dll.  The (hidden) `-select-validator` option has been removed.
+
+#### Notable SPIR-V updates
+
+- Fix unnecessary Int64 requirement when loading Float64
+- Added vk::BufferPointer, see [proposal](https://github.com/microsoft/hlsl-specs/blob/main/proposals/0010-vk-buffer-ref.md) for more details.
+- Implement QuadAny and QuadAll (#7266)
+- Fix -fvk-invert-y (#7447)
+
+#### Shader Model 6.9 Preview
+
+You can now compile shaders to SM 6.9, but this is a preview, so shader hashes will be set to the PREVIEW_BYPASS pattern.
+SM 6.9 shaders will only work with AgilitySDK 1.717.0-preview, a supported preview driver, and use of experimental shader models in developer mode.
+Preview shaders will not be compatible with the SM 6.9 release, or likely even later versions of the SM 6.9 preview.
+
+SM 6.9 Preview Additions:
+
+- Long vectors are allowed in HLSL when targeting shader model 6.9. Vectors up to 1024 elements in length can be loaded from/stored to raw buffers and used in elementwise operations. See the [long vector proposal](https://github.com/microsoft/hlsl-specs/blob/main/proposals/0026-hlsl-long-vector-type.md) for more details.
+- HLSL Vectors are still limited to a maximum of 4 elements when used in certain contexts:
+  - entry function inputs/outputs
+  - parameter, payload, attribute, and node record types for mesh, raytracing, and node shaders
+  - constant buffers (cbuffer), texture buffers (tbuffer), textures and typed buffers
+  - Note: some HLSL elementwise intrinsics do not yet support long vectors in this preview
+- Native vectors of up to 1024 elements are now present in DXIL. This includes vector llvm instructions, load/store, and various elementwise DXIL operations. This may result in smaller DXIL and potentially other performance improvements. See the [dxil vectors proposal](https://github.com/microsoft/hlsl-specs/blob/main/proposals/0030-dxil-vectors.md) for more details.
+- New Linear Algebra (LinAlg) operations (a.k.a. Cooperative Vectors). See the [cooperative vectors proposal](https://github.com/microsoft/hlsl-specs/blob/main/proposals/0029-cooperative-vector.md) and the [HLSL header based API proposal](https://github.com/microsoft/hlsl-specs/blob/main/proposals/0031-hlsl-vector-matrix-operations.md) for more details.
```

Comment:
It's my understanding that linear algebra is a superset of cooperative vectors rather than an aka. Maybe something like:
```suggestion
- Coopertive Vectors, a new set of Linear Algebra (LinAlg) operations. See the [cooperative vectors proposal](https://github.com/microsoft/hlsl-specs/blob/main/proposals/0029-cooperative-vector.md) and the [HLSL header based API proposal](https://github.com/microsoft/hlsl-specs/blob/main/proposals/0031-hlsl-vector-matrix-operations.md) for more details.
```